Signal/src/view controllers/CallViewController.swift

```diff
@@ -506,6 +528,37 @@ class CallViewController: UIViewController, CallDelegate {
 
         default: break
         }
+
+        if callState == .connected {
+            if callDurationTimer == nil {
+                let kDurationUpdateFrequencySeconds = 1 / 20.0
```

Comment:
It was a bit arbitrary.  I just wanted the frame rate to be fast enough that the ticking of seconds would feel smooth and regular.